### DISCUSSION TITLE
I am having problem installing expo-cli, any help on that

### DISCUSSION THREAD

**Discussion by CaptainEboy** [termux/termux-packages#8362] [Q&A]
I've installed git & nodeJs successfully on termux but anytime I use the command, npm install --global expo-cli, I get the error CANNOT LINK EXECUTABLE 'node'.

More info in the attached image below👇

![Uploading Screenshot_20211225-143821_1640540211153.jpg…]()

**Comment by Robinbinu** [comment]
Try reinstalling Nodejs, maybe the target repos are missing in ur installation, see that you  have an uninterrupted data access.

**Comment by CaptainEboy** [comment]
I actually discovered that nodeJs wasn't earlier installed.
However I think the problem came from termux, I had to read the documentation on termux, change the mirror & also upgrade all packages before I could successfully install both nodeJs & git.

I am still yet to successfully install expo-cli as it's taking forever to get installed, so I am still open to suggestions.

Thanks
